packages/psss/package.json

```diff
@@ -30,7 +30,7 @@
     "@material-ui/styles": "^4.9.10",
     "@tupaia/ui-components": "workspace:*",
     "@tupaia/utils": "workspace:*",
-    "axios": "^0.21.0",
+    "axios": "^0.28.0",
```

Comment:
I'd be interested in what tests break with upgrading this in PSSS - might be with fixing things to work with the new version since there are some security issues at play?

Reply:
Turns out it was an interaction between Axios and Jest, which you can imagine a lot of people are having issues with. Managed to find a reasonably tidy workaround

Thanks for the nudge!